src/utils/isCollected.ts

```diff
@@ -3,8 +3,8 @@ type isCollectedArgs = {
   id: string
 }
 
-const isCollected = ({ cards, id }: isCollectedArgs) => {
-  return cards.find((card) => card.id === id)
+const isCollected = ({ cards, id }: isCollectedArgs): boolean => {
+  return cards.find((card) => card.id === id) ? true : false
```

Comment:
```suggestion
  return cards.some((card) => card.id === id)
```

some() is like find() but it'll return a boolean instead of the object/undefined